data/benefits/javascript/aide-logement-etudiant-social.yml

```
@@ -0,0 +1,37 @@
+label: Résidences étudiantes à loyer modéré
+institution: etat
+description: >
+  Certains bailleurs sociaux et associations proposent des logements étudiants à loyer modéré : studios, T2 ou colocations meublées, souvent situés près des campus.
+  Ces résidences sont destinées aux étudiants et apprentis aux revenus modestes et peuvent donner droit aux aides au logement.
+  La demande se fait directement auprès des gestionnaires de ces résidences.
+conditions_generales:
+  - type: age
+    operator: ">="
+    value: 18
+  - type: age
+    operator: <=
+    value: 30
+  - type: difficultes_acces_ou_frais_logement
+    value: true
+conditions:
+  - Effectuer une recherche sur le site <a target="_blank" title="Mon Logement Etudiant - Nouvelle fenêtre" rel="noopener" href="https://monlogementetudiant.beta.gouv.fr/trouver-un-logement-etudiant">Mon Logement Etudiant</a> pour découvrir l'offre de résidences étudiantes à loyer modéré.
```

Alternative plus cohérente qui a sa place dans cette section de conditions :
```suggestion
  - Déterminer les logements à loyer modérés adaptés à ma situation avec <a target="_blank" title="Mon Logement Etudiant - Nouvelle fenêtre" rel="noopener" href="https://monlogementetudiant.beta.gouv.fr/trouver-un-logement-etudiant">Mon Logement Etudiant</a>.
```